code/game/objects/items/devices/scanners.dm

```diff
@@ -647,6 +647,7 @@ BREATH ANALYZER
 /obj/item/device/advanced_healthanalyzer/attack(mob/living/M, mob/living/user)
 	if(!internal_bodyscanner)
 		return
+	user.visible_message("<b>[user]</b> starts scanning \the [M] with \the [src].", SPAN_NOTICE("You start scanning \the [M] with \the [src]."))
```

Comment:
Should be a SPAN_NOTICE and \the [user] for the first part of the message.
Also as a side note, don't use the IC changelog in your PRs. That's what makes it start showing up on newscasters. The changelog .yml is all we need!

Reply:
This doesn't use SPAN_NOTICE as, in the same format as health analysers, the message sent to everyone else is black, while the message the user sees is blue (the notice). The same reasoning was behind not using \the; Health analysers would just say "Monkey (343) runs the scanner over John Smith" (if monkeys can use them, idk), and I wanted to stick to the same format.

Reply:
Also, thanks for the heads up about the IC changelog.

Reply:
Alright, if that's what the health scanners use, forget the SPAN. Still good practice to use \The just in case a special human mob (like "grue") uses it for some reason. Not mandatory, I guess.